cmd/vspd/vspd.go

```diff
@@ -270,6 +275,62 @@ func (v *vspd) checkPurchaseHeights() error {
 	return nil
 }
 
+// checkRevoked ensures that any tickets in the database with outcome set to
+// revoked are updated to either expired or missed.
+func (v *vspd) checkRevoked() error {
+	revoked, err := v.db.GetRevokedTickets()
+	if err != nil {
+		return fmt.Errorf("db.GetRevoked error: %w", err)
+	}
+
+	if len(revoked) == 0 {
+		// Nothing to do, return.
+		return nil
+	}
+
+	v.log.Warnf("Updating %s in revoked status, this may take a while...",
```

Comment:
Adding context for "this may take a while", on my testnet VSP 

```no-highlight
[WRN] VSP: Updating 231 tickets in revoked status, this may take a while...
[DBG] VSP: Scanning 109113 blocks for 231 spent tickets
[INF] VSP: Took 36.742324728s
```